service/src/main/java/uk/gov/hmcts/reform/fpl/enums/UserRole.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public enum UserRole {
+    LOCAL_AUTHORITY {
+        public List<String> getRoles() {
+            return addRoles("caseworker-publiclaw-solicitor");
+        }
+    },
+
+    HMCTS_ADMIN {
+        public List<String> getRoles() {
+            return addRoles("caseworker-publiclaw-courtadmin");
+        }
+    },
+
+    CAFCASS {
+        public List<String> getRoles() {
+            return addRoles("caseworker-publiclaw-cafcass");
+        }
+    },
+
+    GATEKEEPER {
+        public List<String> getRoles() {
+            return addRoles("caseworker-publiclaw-gatekeeper");
+        }
+    },
+
+    JUDICIARY {
+        public List<String> getRoles() {
+            return addRoles("caseworker-publiclaw-judiciary");
+        }
+    };
+
+    public abstract List<String> getRoles();
+
+    private static List<String> addRoles(final String roleToAdd) {
```

Comment:
Would it not be simpler to just save the role of the enum as a private string called `role` or something and then instead of having `addRoles(...)`, `getRoles()` can just be `return new ArrayList<>(Arrays.asList("caseworker", "caseworker-publiclaw", this.role));`

Just seems a bit simpler

Reply:
we could but we will have to repeat `"caseworker", "caseworker-publiclaw"` for every `getRoles(...)` which is what was avoiding